eng/tools/generator/cmd/issue/link/pullRequestLink.go

```diff
@@ -44,6 +45,23 @@ func (l pullRequestLink) Resolve() (ResolveResult, error) {
 	linq.From(files).Select(func(item interface{}) interface{} {
 		return item.(*github.CommitFile).GetFilename()
 	}).ToSlice(&filePaths)
+
+	tspConfig, err := GetTspConfigPathFromChangedFiles(l.ctx, l.client, filePaths)
```

Comment:
There are a few places calling `GetTspConfigPathFromChangedFiles`. Can we call it once then use the return value in other places?